Cargo.toml

```diff
@@ -38,8 +38,8 @@ wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4"
 
 [features]
-default = ["isahc-static-curl"]
-isahc-static-curl = ["isahc/static-curl"]
+default = ["isahc", "isahc-static-curl"]
```

Comment:
Added description to readme.
I'm a little unsure whether to write a code example or not in here, 
because writing something that moves is going to be huge!🤔

https://github.com/meilisearch/meilisearch-rust/pull/459/commits/5993c50552fa5f7a4ca63edf5dc3657b6800ce9b